### DISCUSSION TITLE
How do we use a Release?

### DISCUSSION THREAD

**Discussion by rgl** [spinframework/spin-js-sdk#293] [Q&A] [answered]
How do we use a release from https://github.com/fermyon/spin-js-sdk/releases (e.g. https://github.com/fermyon/spin-js-sdk/releases/tag/v2.2.0)?

Aren't these releases supposed to be delivered as a spin plugin? As such, there should be a, e.g., 2.2.0, version in https://github.com/fermyon/spin-plugins/tree/main/manifests/js2wasm?

**Comment by radu-matei** [comment, marked as answer]
Hey, @rgl, great to see you again!

A few months ago we introduced the v2 of the JS SDK which no longer requires a plugin -- more details in this post https://www.fermyon.com/blog/introducing-the-new-js-sdk and the docs for the new SDK https://developer.fermyon.com/spin/v3/javascript-components#building-and-running-the-template.

TL; DR is that v2 is a breaking change in case you have existing apps built with v1 of the JS SDK.

If you're starting a new app, then using the template brings in the desired version of the SDK.
Updates and new releases are consumed by bumping the version number in package.json.


Let me know if this makes sense.

**Reply by rgl** [reply]
Thank you! I've missed those posts, which I've now read and was able to update my example. :-)
